### ISSUE TITLE
Extend Attribute doesn't extend env?

### ISSUE THREAD

**Issue by LLouice** [sagiegurari/cargo-make#579]
Currently,cargo-make(v0.35.0) run below Makefile.toml
```toml
[tasks.task1]
env = { "Foo" = "foo" }
command = "echo"
args = ["${Foo}"]

[tasks.task2]
extend = "task1"
```
 The task2 extends task1 but gives different result, task1 gives `foo`, task2 gives `${Foo}`.

  It's confusing and seems that task2 does not extend  `task1.env`.

  Is that the default behavior or a bug?

**Comment by sagiegurari** [comment]
Thanks for reporting it. feels like a bug.

**Comment by sagiegurari** [comment]
@LLouice sorry for the late handling, this should now be fixed in branch 0.35.1
Would love you to verify on your end that it resolves your issues.
thanks a lot for reporting this

**Comment by LLouice** [comment]
great! I will try it some hours later and give feedback.thanks for your work!

**Comment by LLouice** [comment]
This bug now be fixed in branch 0.35.1, close it.

**Comment by sagiegurari** [comment]
thanks for checking and for the feedback
